src/js/view/styledComponents/TabPanelStyledComponents.js

```diff
@@ -45,3 +45,22 @@ export const Button = styled.div`
   padding: 5px;
   margin-left: 5px;
 `;
+
+export const Indicator = styled.div`
+  width: 10px;
+  height: 10px;
+  display: inline-block;
+  background-color: lightgreen;
+  border-radius: 50%;
+  position: relative;
+  top: -8px;
+  left: 5px;
+  border: darkgrey solid 1px;
```

Comment:
Maybe darken this border just slightly? (Yes, in css colors grey is darker than darkgrey)
```suggestion
  border: grey solid 1px;
```

Reply:
I see your point, will darken the border.